Bottom: ae275932da43d11f15ec934c737fe2bc4da57ba6
Top:    c83d0f4cd43faddbabd30a82dee78d37606cd436
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-07 14:32:20 -0500

Refresh of all-docker-image-repositories

---
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 7d2c6898..6d374c60 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -58,9 +58,6 @@
     group: tape
     mode: 'u=rw,g=r,o='
 
-- name: copy script
-  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-mantisbt/mantisbt.sh {{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}'
-
 - name: exists - state file
   stat:
     path: '{{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}/restore.date.txt'
@@ -76,6 +73,12 @@
     version: master
     dest: '{{ docker_projects_dir }}/docker-mantisbt'
 
+- name: copy script
+  copy:
+    remote_src: true
+    src: '{{ docker_projects_dir }}/docker-mantisbt/mantisbt.sh'
+    dest: '{{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}/'
+
 - name: build image
   docker_build:
     image_name: '{{ mantisbt_image_name }}'
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 3dda287b..0ca009b7 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -58,9 +58,6 @@
     group: tape
     mode: 'u=rw,g=r,o='
 
-- name: copy script
-  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-mediawiki/mediawiki.sh {{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}'
-
 - name: exists - state file
   stat:
     path: '{{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}/restore.date.txt'
@@ -77,6 +74,9 @@
     version: master
     dest: '{{ docker_projects_dir }}/docker-mediawiki'
 
+- name: copy script
+  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-mediawiki/mediawiki.sh {{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}'
+
 - name: build image
   docker_build:
     image_name: '{{ wiki_image_name }}'
